src/schemas/Stage.yaml

```diff
@@ -0,0 +1,11 @@
+type: object
+required:
+  - id
+  - name
+properties:
+  id:
+    type: string
+    format: uuid
+  name:
+    type: string
+    description: stage name
```

Comment:
iirc every file under `/schemas` should be explicitly set in `/src/schemas/_index.yaml` otherwise you won't have the proper declared structures on typescript side